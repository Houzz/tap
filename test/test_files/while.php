<?php

$x = 1;
while ($x) {}   // error for $x's type

while (true) {
  $y = 2;
}

__tap_check($y, "int|null");
