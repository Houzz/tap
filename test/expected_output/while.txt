4 E_EXPECTED_TYPE
10 E_NAME_UNDEFINED
